webserver/pkgpkr/pkgpkr/settings.py

```diff
@@ -123,10 +123,6 @@
 # Env keys
```

Comment:
Ignore my previous comment, this is what I meant. However, I just want to know why not just leave the exact credentials here without inputting from OS? We can keep a version of `setting.py` locally and modify it by ourselves.